static/styles/settings.css

```diff
@@ -632,11 +632,20 @@ input[type="checkbox"] {
 }
```

Comment:
We also need 
```
border-left: 0px;
border-radius: 0px 4px 4px 0px;

```
for the stream box so that it looks like this:
<img width="375" alt="Screenshot 2021-03-30 at 3 04 02 PM" src="https://user-images.githubusercontent.com/25124304/112967712-2d262400-9169-11eb-830f-a63dd6defe89.png">